### PR TITLE
[SPARK-20005][WEB UI]fix 'There is no "Newline" in UI in describtion'.

### PR DESCRIPTION
**Pull request by guoxiaolongzte**
## What changes were proposed in this pull request?

Before I fix this issue, if the describtion is too long, the content is still displayed on a line, causing the cell to be stretched indefinitely. The UI page is very ugly.
Add a css class, if the overdone is too long, I will automatically intercept the string, with '...' to replace, if the user wants to see more detailed in the word, you can double click to describtion, then more lines show the detail of describtion, then the UI page comparison Regular.
Please refer to the attachment of issue 'SPARK-20005',before_fix_jobs.png, before_fix_stages.png and after_fix_jobs.png, after_fix_stages.png.
issue url:https://issues.apache.org/jira/browse/SPARK-20005.

## How was this patch tested?

manual tests

Please review http://spark.apache.org/contributing.html before opening a pull request.

### CODE REVIEW

File: core/src/main/scala/org/apache/spark/ui/jobs/AllJobsPage.scala [hunk 1]
```diff
@@ -500,7 +500,7 @@ private[ui] class JobPagedTable(
   override def tableId: String = jobTag + "-table"
 
   override def tableCssClass: String =
-    "table table-bordered table-condensed table-striped " +
+    "table table-bordered table-condensed table-striped sortable " +
```
Comment (srowen) [inline core/src/main/scala/org/apache/spark/ui/jobs/AllJobsPage.scala]: It doesn't sound like this just makes it wrap. I'm not sure what the implications are of this?

File: docs/monitoring.md [hunk 2]
```diff
@@ -299,6 +299,7 @@ can be identified by their `[attempt-id]`. In the API listed below, when running
   <tr>
     <td><code>/applications/[app-id]/stages</code></td>
     <td>A list of all stages for a given application.</td>
+    <br><code>?status=[active|complete|pending|failed]</code> list only stages in the state.
```
Comment (srowen) [inline docs/monitoring.md]: Unrelated?

**Review by srowen — CHANGES_REQUESTED** [review]

**Comment by guoxiaolongzte** [comment]
@srowen 
Look at the attachment of 'SPARK-20005', you know what happened.


sorry,this is Unrelated. I merge code is error about docs/monitoring.md.

File: docs/monitoring.md (hunk 2 above)
Comment (guoxiaolongzte) [inline docs/monitoring.md]: sorry,this is Unrelated. I merge code is error about docs/monitoring.md.

**Comment by srowen** [comment]
Ping @guoxiaolongzte to fix or close but i'm not yet clear this does only what you intend.
CC @ajbozarth

**Comment by guoxiaolongzte** [comment]
@ajbozarth

**Comment by guoxiaolongzte** [comment]
Does the PR nobody deal with it?@srowen

**Comment by srowen** [comment]
@guoxiaolongzte I'm not sure what you're asking. It is up to you to fix this pull request, or else close it if you're not going to.
